README.md

```diff
@@ -24,9 +27,107 @@ Other testing frameworks often support fixtures, which bracket a set of tests wi
+| `-T`, `--no-tests`           | Skip running the tests themselves. Useful for running/testing just the set-up and/or clean-up steps.                                                                                                                                       |
+| `-u`, `--test-user` USERNAME | Use the test user account with the given name. See also the `-a` and `-p` options.                                                                                                                                                         |
+| `-v`, `--verbose`            | Print the name of each test as it is run; generally a good idea.                                                                                                                                                                           |
+| `-h`, `--help`               | Print usage information and exit.                                                                                                                                                                                                          |
+| `--version`                  | Print the script version and exit.                                                                                                                                                                                                         |
+
+### Config file syntax ###
+
+Unfortunately, the names of the variables in the config file are not the same as their names on the command line. Below is a translation table and an example config file.
+
+| Command-Line         | Config File   | Default Value |
+|:---------------------|:--------------|:--------------|
+| --add-user           | adduser       | False         |
+| --cilogon            | cilogon       | True          |
+| --dump-output        | dumpout       | False         |
+| --dump-file          | dumpfile      | None          |
+| --extra-repo         | extrarepos    | []            |
+| --exit-on-fail       | exitonfail    | False         |
+| --update-repo        | updaterepos   | []            |
+| --install            | packages      | []            |
+| --manual-run         | manualrun     | False         |
+| --no-cleanup         | skip_cleanup  | False         |
+| --no-print-test-name | printtest     | False         |
+| --password           | password      | vdttest       |
+| --securepass         | securepass    | False         |
+| --update-release     | updaterelease | None          |
+| --tarballs           | tarballs      | False         |
+| --no-tests           | skiptests     | False         |
+| --test-user          | username      | vdttest       |
+| --verbose            | verbose       | False         |
+|                      | backupmysql   | False         |
+|                      | hostcert      | False         |
+|                      | nightly       | False         |
+|                      | selinux       | False         |
```

Comment:
So there's some osg-test configuration options that do not have command line options?

Reply:
Yup, the idea was that we didn't want to expose the more 'advanced' options to users. *shrug*